modules/environment/main.tf

```diff
@@ -1,34 +1,23 @@
-module "environment_context" {
-  source = "../context"
-
-  enabled = true
-
-  context_name          = var.stack_config_name
-  environment_variables = var.environment_values
-}
-
-module "components" {
+module "stacks" {
   source = "../stack"
 
-  for_each = {
-    for k, v in var.components : "${var.stack_config_name}-${k}" => merge({ "component" : k }, v)
+  for_each = { 
+    for k, v in var.components : 
+      format("%s-%s", var.stack_config_name, k) => merge({ "component" : k }, v)
   }
 
-  enabled               = try(each.value.workspace_enabled, true)
-  stack_name            = "${var.stack_config_name}-${each.value.component}"
-  environment_name      = var.stack_config_name
+  enabled               = try(each.value.workspace_enabled, false)
+  stack_name            = each.key
   autodeploy            = coalesce(try(each.value.autodeploy, null), var.autodeploy)
   component_root        = format("%s/%s", var.components_path, try(each.value.custom_component_folder, each.value.component))
```

Comment:
This should probably be `component` now.

Reply:
yeah things are getting a little confusing w/ the naming because our `stack` (e.g. `usw2-sandbox`) and a Spacelift `stack` (e.g. `usw2-sandbox-efs`) are two different things. I'll go ahead and rename it here though.